schema/public.openapi.yaml

```diff
@@ -105,6 +105,18 @@ components:
       maxLength: 10
       example: "5318290"
 
+    PlaybookName:
+      description: Human readable name of the playbook run. Used to present the given playbook run in external systems (Satellite).
+      type: string
+      example: Fix Critical CVEs
+      minLength: 1
+
+    WebConsoleUrl:
+      description: URL that points to the section of the web console where the user find more information about the playbook run. The field is optional but highly suggested.
+      type: string
```

Comment:
Don't forget to also update the RunsFields definition to document these new values

Reply:
done! thanks! :+1: